File: 001-100/001/multiplesOf3or5.js
function multiplesOf3or5(){
  let n = document.getElementById("inputN").value;
  let sum = 0;
  for(let i = 0; i < n; i++){
    if(i % 3 == 0 || i % 5 == 0) { sum++; }
  }
  let answer = `Sum of multiples of 3 or 5 below ${n}<br>${sum}`;
  document.getElementById("answer").innerHTML = answer;
}

File: 001-100/001/multiplesOf3or5.py
def multiplesOf3or5(n):
    sum = 0
    for i in range(0, int(n)):
        if(i % 3 == 0 or i % 5 ==0):
            sum += i
    return sum

if __name__ == "__main__":
    n = input("Enter an integer n: ")
    print("\nSum of multiples of 3 or 5 below {:,}".format(int(n)))
    print("{:,}".format(multiplesOf3or5(n)))
